Bio/Affy/CelFile.py

```diff
@@ -185,7 +318,3 @@ def read(handle):
         else:
             continue
     return record
-
-if __name__ == "__main__":
-    from Bio._utils import run_doctest
-    run_doctest()
```

Comment:
Did you remove this deliberately? TravisCI etc will run this via the listing inside ``run_tests.py`` but having this ``__main__`` trick is usually very helpful for checking the doctests during development.